### DISCUSSION TITLE
How to withdraw the unpublishing site command

### DISCUSSION THREAD

**Discussion by Petrelli** [community/community#33244] [Actions] [answered]
I clicked the unpublishing site button in the page settings, and I fond there are no options to withdraw this command. Does it mean I cannot publish any website, e.g., personal homepage on GitHub.io?

**Comment by jfullstackdev** [comment, marked as answer]
what do you mean by that ? there is a new UI to unpublish site

![image](https://user-images.githubusercontent.com/47092464/190637906-a8500d14-fc18-43a3-8862-e45bfbdfb9ca.png)

**Reply by Petrelli** [reply]
I mean can I go back to the UI you posted after I clicked the "Unpublish site" button?

**Reply by jfullstackdev** [reply]
this is the view after unpublishing it 

![image](https://user-images.githubusercontent.com/47092464/191698616-8a9a54b6-d399-49a9-abed-fb6a50f3dd56.png)
